### PR TITLE
Add ALICE cluster to profiles

### PR DESCRIPTION
**Pull request by bbartholdy**
Addition of the ALICE cluster at Leiden University to profiles

### CODE REVIEW

File: docs/alice.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+Before running the pipeline you will need to load Java and Nextflow. You can do this by including the commands below in your SLURM/sbatch script:
+
+```bash
+## Load Java and Nextflow environment modules
+module load Singularity/3.6.1-Go-1.14
+module load Nextflow/21.03.0
+module load Java/11.0.2
+```
+
```
Comment (jfy133) [inline docs/alice.md:15]:
```suggestion

We also highly recommend specifying a location of a cache directory to store singularity images (so you re-use them across runs, and not pull each time), by specifying the location with the `$NXF_SINGULARITY_CACHE_DIR` bash environment variable  in your `.bash_profile` or `.bashrc`.

```

**Review by jfy133 — APPROVED** [review]
LGTM, one comment for the docs.

File: docs/alice.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+We also highly recommend specifying a location of a cache directory to store singularity images (so you re-use them across runs, and not pull each time), by specifying the location with the `$NXF_SINGULARITY_CACHE_DIR` bash environment variable  in your `.bash_profile` or `.bashrc`.
+
+All of the intermediate files required to run the pipeline will be stored in the `work/` directory. It is recommended to delete this directory after the pipeline has finished successfully because it can get quite large, and all of the main output files will be saved in the `results/` directory anyway.
+The config contains a `cleanup` command that removes the `work/` directory automatically once the pipeline has completeed successfully. If the run does not complete successfully then the `work/` dir should be removed manually to save storage space.
+
+This configuration will automatically choose the correct SLURM queue (short,medium,long) depending on the time required by each process. If there are
```
Comment (jfy133) [inline docs/alice.md:21]:
```suggestion
This configuration will automatically choose the correct SLURM queue (short,medium,long) depending on the time required by each process. If there are
```

File: conf/alice.config [hunk 3]
```diff
@@ -0,0 +1,39 @@
+params {
+      config_profile_name = 'ALICE'
+      config_profile_description = 'Profile for use on Academic Leiden Interdisciplinary Cluster Environment (ALICE).'
+      config_profile_contact = 'Bjorn Peare Bartholdy (@osteobjorn)'
+      config_profile_url = 'https://wiki.alice.universiteitleiden.nl/index.php?title=ALICE_User_Documentation_Wiki'
```
Comment (maxulysse) [inline conf/alice.config:5]:
```suggestion
      config_profile_url = 'https://wiki.alice.universiteitleiden.nl/'
```

File: conf/alice.config [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+process {
+  executor = 'slurm'
+  queue = { task.time < 3.h ? 'cpu-short' : task.time < 24.h ? 'cpu-medium' : 'cpu-long' }
+}
+
+singularity {
+  enabled = true
+  autoMounts = true
+}
```
Comment (maxulysse) [inline conf/alice.config:19]: No cache directory specified for Singularity?

**Review by maxulysse — COMMENTED** [review]
URL for the profile can be simpler, apart from that only suggestion is to set up the singularity cache dir for the whole cluster, unless there's a reason to ask users to have it set up in their own .bashrc

File: conf/alice.config (hunk 4 above)
Comment (bbartholdy) [reply conf/alice.config:19]: As far as I'm aware there's no shared storage on the cluster. The 'shared data' are stored under each user's username and sym-linked to the home directory

Comment (jfy133) [reply conf/alice.config:19]: Often doesn't work for large multi-group clusters, same at a lot of German HPCs. Have to set it per person

**Comment by jfy133** [comment]
All good now!

**Pull request merged**